meshroom/ui/qml/main.qml

```diff
@@ -46,6 +46,8 @@ ApplicationWindow {
     SystemPalette { id: activePalette }
     SystemPalette { id: disabledPalette; colorGroup: SystemPalette.Disabled }
 
+    property string imagesFolder: ""
```

Comment:
Should use a `url` property instead of `string` since it represents a location on the file system

Reply:
Done.